oak_restricted_kernel/src/lib.rs

```diff
@@ -386,10 +386,20 @@ pub fn start_kernel(info: &BootParams) -> ! {
         }
     };
 
-    let (derived_key, restricted_kernel_dice_data) =
-        oak_restricted_kernel_dice::attest_application(stage0_dice_data, &application_bytes);
-
     log::info!("Binary loaded, size: {}", application_bytes.len());
+
+    let app_digest = oak_restricted_kernel_dice::measure_app_digest(&application_bytes);
+
+    log::info!(
+        "Binary hash: {}",
```

Comment:
(nit) for consistency?

```suggestion
        "Application digest: {}",
```

Reply:
Could you also print what hash function is used? is it sha2-256? If so, perhaps we should rename `measure_app_digest` to include the hash type itself, to avoid confusion if / when we support other ones? In fact perhaps rename `AppDigest` to `AppDigestSha2_256` or something like that?

Reply:
+1